sdk/messaging/azservicebus/internal/atom/errors.go

```diff
@@ -6,10 +6,15 @@ package atom
 import (
 	"fmt"
 	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 )
 
-func NewResponseError(inner error, resp *http.Response) error {
-	return ResponseError{inner, resp}
+func NewResponseError(resp *http.Response) error {
+	return &azcore.ResponseError{
```

Comment:
Just confirming there's no useful error code to return?

Reply:
From what I could see the codes are just free form string messages. So 'no', but they will show up in the .Error() since they're in the body.